frontend/src/pages/Admin/index.js

```diff
@@ -40,6 +41,9 @@ function Admin() {
         <NavLink activeClassName="usa-button--active" className="usa-button" to="/admin/email">
           Email
         </NavLink>
+        <NavLink activeClassName="usa-button--active" className="usa-button" to="/admin/training-reports">
```

Comment:
Should this be behind the same feature flag used below?

https://github.com/HHS/Head-Start-TTADP/blob/383fd87c6791aa191783cf61c4d02bb8112073fd/frontend/src/App.js#L266-L270